tokio/src/coop.rs

```diff
@@ -127,10 +137,14 @@ cfg_rt_threaded! {
 
 cfg_blocking_impl! {
     /// Forcibly remove the budgeting constraints early.
-    pub(crate) fn stop() {
+    ///
+    /// Returns the remaining budget
+    pub(crate) fn stop() -> Budget {
```

Comment:
Take it or leave it: we might want to have this just return a guard that puts back the previous budget, and then the worker's `Reset` guard can just own that. If we need to stop budgeting temporarily elsewhere, could be handy. 

On the other hand, this doesn't really matter.

Reply:
It could be done this way, but it would make the worker's block_in_place `Reset` more complicated.

Given that it is internal right now, I am leaning towards punting.

Reply:
yup, fair, just a thought.